README.md

```diff
@@ -63,3 +63,46 @@ Override these rules in `tslint.json`:
 }
 ```
 
+## Inline Linting in Text Editors
+
+
+`tslint-config-shopify` comes with a few rules that require type-checking to be enabled, however `tslint` uses a [private API](https://github.com/palantir/tslint/issues/1445#issuecomment-236330061) to resolves rules that require `type-checking`.
+For this reason, text editors will fail when trying to resolve rules that require `type-checking`.
+
+To resolve this issue, turn off the rules that require type-checking in your `tslint.json`
+```
+{
+  "extends": ["tslint-config-shopify"],
```

Comment:
I find it more useful to point your `extends` at the "full" config in your repo, so that any rule changes for your repo only need to be made in one place.